MyLibrary/MyLibraryApp.API/Controllers/BooksController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MyLibraryApp.Application.Models.Books;
+using MyLibraryApp.Application.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyLibraryApp.API.Controllers
+{
+    [ApiController]
+    [Route("Books")]
+    public class BooksController :ControllerBase
+    {
+        private readonly IBookService bookService;
+        public BooksController(IBookService bookService)
+        {
+            this.bookService = bookService;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetAllBooksAsync()
+        {
+            var response = await bookService.ReadAllBooksAsync();
```

Comment:
`ReadAllBooksAsync` should be `GetAllBooksAsync` as all the other get methods in this controller use the word `get` in their name. Remember: consistency is key when we're coding.